src/templates/admin/review/in_review.html

```diff
@@ -224,6 +224,14 @@ <h2>Actions</h2>
                         <li><a data-open="summarymodal"><i class="fa fa-server action-icon">&nbsp;</i>View Metadata</a>
                         </li>
                         <li><a href="{% url 'document_management' article.pk %}?return={{ request.path }}"><i class="fa fa-file-archive-o">&nbsp;</i>Document Management</a></li>
+                    {% if article.is_accepted %}
+                        <li>
+                            <a href="{% url 'move_to_next_workflow_element' article.pk %}">
+                                <i class="fa fa-arrow-circle-right action-icon">&nbsp;</i>
```

Comment:
As discussed, swap this for a <form method="POST" action="{% url '' %}">

Reply:
Thanks for catching this. I've implemented POST as well as a current stage check.